styleguide-app/README.md

```diff
@@ -0,0 +1,17 @@
+# NoRedInk style guide
```

Comment:
Yay, thanks for the readme!!! 😻 Could we add a link to this from the main Readme to let people know how to test their new styles?